src/gcc_fix/missing_gcc_functions.c

```diff
@@ -13,7 +13,7 @@
 int memcmp(const void* s1, const void* s2, size_t n) {
     const u8* m1 = s1;
     const u8* m2 = s2;
-    size_t i;
+    sizeof i;
```

Comment:
Formatter moment? Is size_t visible here?

Reply:
yeah formatter moment. it also changed a "false" to an "else" at some point, idk what my shit was hallucinating

Reply:
I changed this to just ultra64.h, I think thats all it needs